src/js/views/globals/search/patient-search.scss

```diff
@@ -0,0 +1,78 @@
+.patient-search__modal {
+  height: 420px;
+}
+
+.patient-search__modal-body {
+  height: 100%;
+}
+
+.patient-search__picklist {
+  background-color: transparent;
+  box-shadow: none;
+  height: 100%;
+  max-height: 100%;
+  position: relative;
+  width: auto;
+
+  .picklist__fixed-heading {
```

Comment:
does this need to be nested?

Reply:
Right now, yes in order to avoid affecting other picklists outside this modal. But if I can figure out the template thing you mentioned [here](https://github.com/RoundingWell/care-ops-frontend/pull/208/files#r370335140), we may not need it.